app/_how-tos/create-an-insomnia-plugin.md

```diff
@@ -4,13 +4,14 @@ content_type: how_to
 
 products:
 - insomnia
-
+breadcrumbs: 
+  - /insomnia/plugins/
 tags:
 - plugins
 
 tldr:
   q: How do I create a new Insomnia plugin?
-  a: Create a NodeJS module in the Insomnia plugins directory and add it to Insomnia. To publish on the [Plugin Hub](https://insomnia.rest/plugins), publish it as unscoped public package to the npm public registry.
+  a: Create a NodeJS module in the Insomnia plugins directory and add it to Insomnia. To publish on the [Plugin Hub](https://insomnia.rest/plugins), publish it as an un-scoped public package to the npm public registry.
```

Comment:
```suggestion
  a: Create a NodeJS module in the Insomnia plugins directory and add it to Insomnia. To publish on the [Plugin Hub](https://insomnia.rest/plugins), publish it as an unscoped public package to the npm public registry.
```
It's "unscoped" according to npm: https://docs.npmjs.com/creating-and-publishing-unscoped-public-packages